tests/testdata/bootstrap_tmpl.json

```diff
@@ -305,44 +197,22 @@
                     {"name": "envoy.router"
                     }
                   ],
-                  "route_config": {
-
+                  "rds": {
+                      "route_config_name": "80",
```

Comment:
how can it be port 80's route config? if I set http proxy, can't I do http://foo.com:1111 and still use the proxy on a different port?

Reply:
Well, yes - for now I'm testing with the routes for port 80, one step at a time :-)
For full proxy we'll need to construct an "all" rule.

The point of this PR is to verify some RDS works, and that proxy can be used for tests.